doc/requirements.rst

```diff
@@ -99,6 +99,10 @@ Requirements for mlx.traceability
 .. item:: RQT-DUMMY Dummy requirement that is not covered by a test
     :fulfilled_by: DESIGN-ATTRIBUTES DESIGN-ITEMIZE
 
+.. item:: RQT-DUMMY_CHILD Child of the dummy requirement that is not covered by a test
+    :depends_on: RQT-DUMMY
+.. :validated_by: ITEST-DUMMY_CHILD
```

Comment:
Hm, I assume this is to highlight that there is no `validated_by` relation?

Reply:
No. I hope this change clarifies things: c9f9916d8189981636ed802fe4b9d72f07f03cab